api-doc/Elevate-Project.postman_collection.json

```diff
@@ -0,0 +1,2439 @@
+								"list"
+							],
+							"query": [
+								{
+									"key": "page",
+									"value": "1"
+								},
+								{
+									"key": "limit",
+									"value": "5"
+								},
+								{
+									"key": "search",
+									"value": ""
+								}
+							]
+						}
+					},
+					"response": []
+				},
+				{
+					"name": "join",
+					"protocolProfileBehavior": {
+						"disableBodyPruning": true
+					},
+					"request": {
+						"method": "GET",
+						"header": [
+							{
+								"key": "X-auth-token",
+								"value": "{{userToken}}"
+							},
+							{
+								"key": "internal-access-token",
+								"value": "{{internal_access_token}}"
+							}
+						],
+						"body": {
+							"mode": "raw",
+							"raw": "{\n      \"externalId\" : \"PROGID01\",\n      \"name\" : \"DCPCR School Development Index 2018-19\",\n      \"description\" : \"DCPCR School Development Index 2018-19\",\n      \"isDeleted\" : false,\n      \"resourceType\" : [ \n          \"program\"\n      ],\n      \"language\" : [ \n          \"English\"\n      ],\n      \"keywords\" : [],\n      \"concepts\" : [],\n      \"userId\":\"a082787f-8f8f-42f2-a706-35457ca6f1fd\",\n      \"imageCompression\" : {\n          \"quality\" : 10\n      },\n      \"components\" : [ \n          \"5b98fa069f664f7e1ae7498c\"\n      ],\n      \"scope\" : {\n          \"entityType\" : \"state\",\n          \"entities\" : [\"bc75cc99-9205-463e-a722-5326857838f8\",\"8ac1efe9-0415-4313-89ef-884e1c8eee34\"],\n          \"roles\" : [\"HM\"]\n      },\n      \"requestForPIIConsent\" : true\n}\n",
+							"options": {
+								"raw": {
+									"language": "json"
+								}
+							}
+						},
+						"url": {
+							"raw": "{{baseUrl}}/project/v1/programs/join/665eb764cbf5d12ba882722f",
+							"host": [
+								"{{baseUrl}}"
+							],
+							"path": [
+								"project",
+								"v1",
+								"programs",
+								"join",
+								"665eb764cbf5d12ba882722f"
+							]
+						}
+					},
+					"response": []
+				},
+				{
+					"name": "removeEntitiesInScope",
+					"request": {
+						"method": "POST",
+						"header": [
+							{
+								"key": "X-auth-token",
+								"value": "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJkYXRhIjp7ImlkIjoxLCJuYW1lIjoiUHJpeWFua2EiLCJzZXNzaW9uX2lkIjoyNjQsIm9yZ2FuaXphdGlvbl9pZCI6MSwicm9sZXMiOlt7ImlkIjo3LCJ0aXRsZSI6Im9yZ19hZG1pbiIsInVzZXJfdHlwZSI6MSwic3RhdHVzIjoiQUNUSVZFIiwib3JnYW5pemF0aW9uX2lkIjoxLCJ2aXNpYmlsaXR5IjoiUFVCTElDIn0seyJpZCI6NSwidGl0bGUiOiJtZW50ZWUiLCJ1c2VyX3R5cGUiOjAsInN0YXR1cyI6IkFDVElWRSIsIm9yZ2FuaXphdGlvbl9pZCI6MSwidmlzaWJpbGl0eSI6IlBVQkxJQyJ9LHsiaWQiOjgsInRpdGxlIjoicHJvZ3JhbV9kZXNpZ2luZXIiLCJ1c2VyX3R5cGUiOjAsInN0YXR1cyI6IkFDVElWRSIsIm9yZ2FuaXphdGlvbl9pZCI6MSwidmlzaWJpbGl0eSI6IlBVQkxJQyJ9LHsiaWQiOjksInRpdGxlIjoicHJvZ3JhbV9jcmVhdG9yIiwidXNlcl90eXBlIjowLCJzdGF0dXMiOiJBQ1RJVkUiLCJvcmdhbml6YXRpb25faWQiOjEsInZpc2liaWxpdHkiOiJQVUJMSUMifV19LCJpYXQiOjE3MTk5OTY4ODYsImV4cCI6MTcxOTk5ODY4Nn0.MXgAlooVZv--RaVUoInyCqSueab-E2IH9UBYXQQ2EnY"
```

Comment:
@MallanagoudaB do not pass actual token. you should only pass variable from environment and set the value for that in environment as dummy one. check for same changes in all apis